### ISSUE TITLE
Suggestion: Outside links open in a new window

### ISSUE THREAD

**Issue by artlogic** [allhandsactive/site#32]
The only suggestion is to have the links that link outside of our sit to open in a new page/tab. That way we have more on site time, which is good for the google analytics.

**Comment by onerandomusername** [comment]
I agree, it would be nice to have this feature, so we better keep member retention.
